src/components/app/DebugWarning.js

```diff
@@ -0,0 +1,35 @@
+// @flow
```

Comment:
Can you add the license comment at the top of this file? You can copy paste it from one of the files.